### DISCUSSION TITLE
"Calculating stats for this user. Check back later."

### DISCUSSION THREAD

**Discussion by ChenchenZheng** [athul/waka-readme#139] [Q&A] [answered]
Hello, it has been two days that I tried to make waka-readme work on my repo. I keep on receiving this error: 

> Exception: Query 'https://wakatime.com/api/v1/users/current/all_time_since_today?api_key=***' failed to run by returning code of 202: {'data': {'is_up_to_date': False, 'range': {'start': '2023-07-21T22:00:00Z', 'start_date': '2023-07-22', 'start_text': 'Yesterday', 'end': '2023-07-23T21:59:59Z', 'end_date': '2023-07-23', 'end_text': 'Today', 'timezone': 'Europe/Paris'}, 'timeout': 15, 'percent_calculated': 0, 'total_seconds': 0, 'text': '0 secs', 'decimal': '0.00', 'digital': '0:00'}, 'message': 'Calculating stats for this user. Check back later.'}

My username on waka-readme is `pinou_boss`.

Do you have any idea how can I fix this? Thanks a lot for your help!

**Comment by ChenchenZheng** [comment]
It seems like we should push at minimum a commit to have sth to display on waka-readme dashboard, then the job runs successfully finally

**Comment by nandgator** [comment, marked as answer]
Hi @ChenchenZheng, `HTTP 202` is `'accepted'`, which means you've requested a big chunk of data which the server is processing.

By the way you're using a different project https://github.com/anmol098/waka-readme-stats.

Feel free to checkout https://github.com/athul/waka-readme, which follows a minimalist approach and has a retry request mechanism, if the response isn't `HTTP 200` or `'ok'`.
